internal/pkg/agent/application/upgrade/rollback.go

```diff
@@ -238,3 +242,22 @@ func restartAgent(ctx context.Context, log *logger.Logger, c client.Client) erro
 	close(signal)
 	return nil
 }
+
+func makeBaseWatchCmd(agentExecutable string, rollbackWindow time.Duration) *exec.Cmd {
+	cmdArgs := []string{
+		watcherSubcommand,
+		"--path.config", paths.Config(),
+		"--path.home", paths.Top(),
+	}
+
+	if rollbackWindow > 0 {
+		cmdArgs = append(cmdArgs, "--rollback-window", fmt.Sprintf("%.fs", rollbackWindow.Seconds()))
```

Comment:
Is there a specific reason why this needs to be printed as seconds instead of just using [Duration.String()](https://pkg.go.dev/time#Duration.String) ?
I think that is a more readable/compact format than a float for seconds :thinking:

Reply:
I can't recall why I did this but I can't think of any reasons seconds is necessarily better.  Changed to use Duration.String() in 9f87950cdaf4702a6e1f8ec51a77e1a427dc8fc7.